xlators/cluster/afr/src/afr.c

```diff
@@ -987,7 +987,7 @@ struct volume_options options[] = {
      .type = GF_OPTION_TYPE_INT,
      .min = 1,
      .max = 1024,
-     .default_value = "1",
+     .default_value = "8",
```

Comment:
Can you also update the description string to explicitly mention that one block is 128KiB in size?